core/src/test/scala/org/apache/spark/ContextCleanerSuite.scala

```diff
@@ -52,6 +51,7 @@ abstract class ContextCleanerSuiteBase(val shuffleManager: Class[_] = classOf[Ha
     .setAppName("ContextCleanerSuite")
     .set("spark.cleaner.referenceTracking.blocking", "true")
     .set("spark.cleaner.referenceTracking.blocking.shuffle", "true")
+    .set("spark.cleaner.referenceTracking.cleanCheckpoints", "true")
```

Comment:
This is a context cleaner suite, and its purpose is exactly to test cleaning up of various things including checkpoints. It's natural to turn this one on here since we use it in multiple tests in the suite. It's a minor point, but I really think we should just enable this here, but if you feel strongly about this I'll revert the changes.